src/middleware/builtin.ts

```diff
@@ -251,22 +251,6 @@ function matchesPattern(pattern: string | RegExp, candidate: string): boolean {
   return pattern.test(candidate);
 }
 
-/**
```

Comment:
Removed due to reverted commit included in history after bad rebase. The changed version should match `main`. To verify: [L254](https://github.com/slackapi/bolt-js/blob/237a503d1e19a8e40ca9ed130dacab1be9752d53/src/middleware/builtin.ts#L254)